packages/webpack/lib/utils/forkedCompiler.js

```diff
@@ -8,9 +8,9 @@ const { createCompiler } = require('./compiler');
 process.on('message', (message) => {
   if (message.name !== 'start') return;
 
-  const { buildConfigArgs, overrides, options } = message;
-  const webpackConfig = configure(overrides, options).getBuildConfig(
-    ...buildConfigArgs
+  const { webpackConfigArgs, overrides, options } = message;
+  const webpackConfig = configure(overrides, options).getWebpackConfig(
```

Comment:
`hops start` throws the error `Invariant violation: getWebpackConfig(): Received invalid name string` for me here for the `node`-build, when explicitly running on commit `3f5ddaf`.